### ISSUE TITLE
Electron applications does not read if aria-atomic is false.

### ISSUE THREAD

**Issue by Tanuja-Kulal** [FreedomScientific/VFO-standards-support#89]
### Summary

I work on electron application which renders in chrome. Jaws does not read text in live region unless **aria-atomic = true** is mentioned. There are few logs which i want to read with aria-live="polite" and aria-atomic="false" settings. In this scenario JAWS does not read live region but NVDA does.

Here is version details of my app.
> Electron: 2.0.0
> Chromium: 61.0.3163.100
> Node.js 8.9.3
> JAWS: 2018.1805.33 ILM

Repro steps 
> 1. clone https://github.com/Tanuja-Kulal/ElectronApp  
> 2. follow instructions in ReadMe.md
> 3. Run electron application 
> 2. click on "display random number" button

### Expected result

> The new numbers should read out

### Actual result

> The numbers are not read.

#### More information
Toggle aria-atomic flag in index.html and try. JAWS reading live changes.

#### JAWS version and build number
 2018.1805.33 ILM
 
#### Operating System and version
Windows 10 Enterprise 
[electron-quick-start.zip](https://github.com/FreedomScientific/VFO-standards-support/files/2091370/electron-quick-start.zip)

10.0.17134 Build 17134

#### Browser and version:
> Electron: 2.0.0
> Chromium: 61.0.3163.100
> Node.js 8.9.3

**Comment by BenKeyFSI** [comment]
The entire contents of ReadMe.md are as follows.

# ElectronApp

In other words, the instructions are missing.

**Comment by Tanuja-Kulal** [comment]
updated readme

**Comment by BenKeyFSI** [comment]
I am able to reproduce this. I have determined that it is a JAWS bug. Next I will need to consult another JAWS developer for information on how to resolve this issue.

**Comment by BenKeyFSI** [comment]
The bug does not occur when the same web page is opened in Google Chrome. Now to figure out why there is a difference in behavior.

**Comment by BenKeyFSI** [comment]
I have determined the cause of the bug. To be more precise, I now know why the bug occurs in Electron but not in Google Chrome.

Google Chrome supports the IAccessibleText::get_newText. Electron does not.

**Comment by Tanuja-Kulal** [comment]
What is the next steps on this bug?

**Comment by BenKeyFSI** [comment]
I have reported my conclusions to my superiors.

Since the root cause of the bug is that Electron does not support the IAccessibleText::get_newText function while Google Chrome does, we have concluded that this is an Electron bug. I suggest that you file a bug with the developers of Electron.
